components/resolved_ts/src/endpoint.rs

```diff
@@ -379,7 +379,10 @@ where
                 resolver_status,
                 ..
             } = observe_region;
-            info!(
+
+            // register/deregister will fail due to merging/spliting blocked when disk full.
+            // and will generate so much log. so change log level from info to debug.
+            debug!(
```

Comment:
It's necessary to confirm with the author whether it's ok to do this change or not. You can get the author of the code with `git blame`.

Reply:
Ok, I'll check it.